src/modules/shared/pages/innovation/export-requests/export-request-info.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { CoreComponent } from '@app/base';
+
+import { InnovationExportRequestStatusEnum } from '@modules/stores/innovation/innovation.enums';
+
+import { InnovationExportRequestInfoDTO } from '@modules/shared/services/innovations.dtos';
+import { InnovationsService } from '@modules/shared/services/innovations.service';
+
+
+@Component({
+  selector: 'shared-pages-innovation-export-request-info',
+  templateUrl: './export-request-info.component.html'
+})
+export class PageInnovationExportRequestInfoComponent extends CoreComponent implements OnInit {
+
+  innovationId: string;
+  requestId: string;
+  innovationRequest?: InnovationExportRequestInfoDTO;
+
+  // Flags
+  isInnovatorType: boolean;
+  isLeadershipTeamType: boolean;
+
+  constructor(
+    private activatedRoute: ActivatedRoute,
+    private innovationsService: InnovationsService
+  ) {
+
+    super();
+    this.setPageTitle('Permission request');
+    this.setBackLink();
+
+    this.innovationId = this.activatedRoute.snapshot.params.innovationId;
+    this.requestId = this.activatedRoute.snapshot.params.requestId;
+
+    this.isInnovatorType = this.stores.authentication.isInnovatorType();
+    this.isLeadershipTeamType = this.stores.authentication.isAssessmentType() || this.stores.authentication.isAccessorType();
+
+  }
+
+  ngOnInit(): void {
+
+    this.innovationsService.getExportRequestInfo(this.innovationId, this.requestId).subscribe({
+      next: response => {
+        this.innovationRequest = response;
+        this.setPageStatus('READY');
+      },
+      error: () => {
+        this.setPageStatus('ERROR');
+        this.setAlertUnknownError();
+      }
+    });
+
+  }
+
+  updateExportRequestStatus(status: keyof typeof InnovationExportRequestStatusEnum) {
+
+    this.innovationsService.updateExportRequestStatus(this.innovationId, this.requestId, { status }).subscribe({
+      next: () => {
+
+        if (status === InnovationExportRequestStatusEnum.APPROVED) {
+          this.setRedirectAlertSuccess(`You've approved this request`, { message: 'You have given your permission to share or use your innovation record outside of the service for the reason given by the organisation. The organisation will be notified.', width: '2.thirds' });
```

Comment:
It doesn't match the copy from Figma, is it on purpose?
![image](https://github.com/nhsengland/innovation-service-transactional-frontend/assets/115171210/1d4ccf6f-9eb9-4573-9954-ffb06795a25c)